watcher.py

```diff
@@ -42,6 +53,8 @@ def execute(self, command):
             sys.stdout.flush()
             out += line
         proc.wait()
+        if not suppress and proc.returncode != 0:
+            raise subprocess.CalledProcessError(proc.returncode, command, out)
```

Comment:
I'm curious. What's the purpose of having a `suppress` here? We don't seem to use this option anywhere?